src/Nethermind/Nethermind.Config/ExitCodes.cs

```diff
@@ -0,0 +1,25 @@
+﻿//  Copyright (c) 2021 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+//
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+//
+
+namespace Nethermind.Config;
+
+public static class ExitCodes
+{
+    public const int NoEngineModule = 1;
+
+    public const int NoDownloadOldReceiptsOrBlocks = 2;
```

Comment:
I would divide the errors in some numbers, for example configuration errors would be between 1_000 and 1_999, wouldn't go from 1 :)